Projects/App/Sources/Generic/AppFlow.swift

```diff
@@ -95,9 +102,11 @@ extension EmbarkOnboardingFlow {
         case .login:
             return Presentation(
                 Login(),
-                style: .detented(.medium, .large),
-                options: [.allowSwipeDismissAlways, .defaults]
-            )
+                style: .detented(.large),
```

Comment:
it should be `.detented(.medium)` for Sweden. I guess we should use `preferredPresentationStyle` and just `.default`.